python/ql/test/library-tests/ApiGraphs/py3/test_captured_inheritance.py

```diff
@@ -0,0 +1,25 @@
+from foo import A, B
+
+def func():
+    if cond():
+        class Foo(A): pass
+    else:
+        class Foo(B): pass
+
+    class Bar(A): pass
+    class Bar(B): pass
+
+    class Baz(A): pass
+    
+    def other_func():
+        print(Foo)  #$ use=moduleImport("foo").getMember("A").getASubclass() use=moduleImport("foo").getMember("B").getASubclass()
+        # On the next line, we wish to express that it is not possible for `Bar` to be a subclass of `A`.
+        # However, we have no "true negative" annotation, so we use the MISSING annotation instead.
+        # (Normally, "true negative" is not needed as all applicable annotations must be present,
+        # but for type tracking tests, this would be excessive.)
```

Comment:
A bit of a NIT, but these are the API graph tests, not the type-tracking tests :shrug:

```suggestion
        # (Normally, "true negative" is not needed as all applicable annotations must be present,
        # but these API graph tests work differently, since having all results recorded in annotations 
        # would be excessive)
```